client/app/sidenav/employee/employee.controller.js

```diff
@@ -40,10 +67,11 @@
 			}
 		}
 		
-		$scope.employeeList = EmployeeService.query();
+		$scope.employeeList = EmployeeFact.employeeList;
```

Comment:
you should use this and not scope. replace $scope with this and add the controller (this.controller.employeeList)